usaspending_api/etl/es_transaction_mapping.json

```diff
@@ -249,12 +244,7 @@
         },
 
         "recipient_location_country_code": {
-          "type": "text",
-          "fields": {
-            "keyword": {
-              "type": "keyword"
-            }
-          }
```

Comment:
Previously we had discussed the merits of having keyword as a field rather than changing type. What changed?

Reply:
Tony said so

Reply:
This is an open source project, whereas unfortunately (perhaps fortunately) Tony's brain is not. For my own education if nothing else, can we note why this change was wrong to make? "We" can be Tony if needed.

Reply:
`it's returning the value in .keyword which is identical to the default and misses the original purpose of searching exact matches on keyword.` -according to Tony

Reply:
To close the loop: the original purpose was to change the type from `text` to `keyword` since the queries on those fields are only exact match and keyword type should increase performance.

Another option would be to create the keyword subtype and then use the .keyword on the field for exact match queries. IIRC we leaned towards not subtyping unless we had the immediate need for both types